answerking_app/models/serializers.py

```diff
@@ -6,17 +6,16 @@
     MinValueValidator,
     RegexValidator,
 )
-from rest_framework import serializers
-from rest_framework.generics import get_object_or_404
+from rest_framework import serializers, status
 
 from answerking_app.models.models import (
     Category,
-    Item,
+    Product,
     Order,
     OrderLine,
-    Status,
 )
-from answerking_app.utils.model_types import ItemType
+from answerking_app.utils.get_object_or_400 import get_object_or_400
```

Comment:
this function needs a better naming.

Reply:
looks like that you have defined it to works with just products, get_products_or_404?